src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Calculator />
   );
 }
```

Comment:
- [ ] As it is specified in the project requirements, components should be class-based including the App.js component. KIndly implement this change to meet all the project requirements.

![image](https://user-images.githubusercontent.com/86318284/211611495-6fe24d2d-6570-4753-a013-c45e0663c340.png)

Reply:
well resolved! ✔️